cmd/utils.go

```diff
@@ -95,9 +96,103 @@ func splitStr(path, sep string, n int) []string {
 	return splits
 }
 
+// Parse scheme format, supported formats are
+// - "http"
+// - "http+s3v2"
+// - "http+s3v4"
```

Comment:
I think we can remove `http+s3v4` i.e. we can treat it as default.  This way we have four schemes `http`, `http+s3v2`,` https` and `https+s3v2` only

Reply:
this is just an example @balamurugana

Reply:
If you say `http+s3v4` is not a valid scheme, then we would need to remove from example as well.  If supported, my recommendation is to remove the support.

Reply:
No we don't need to parseScheme is generic @balamurugana